src/ifp/src/InitFloorplan.cc

```diff
@@ -227,7 +230,12 @@ void InitFloorplan::initFloorplan(
       }
       makeHybridRows(base_site, sites_by_name, snapped_core);
     } else {
-      makeUniformRows(base_site, sites_by_name, snapped_core, row_parity);
+      makeUniformRows(
+          base_site,
+          sites_by_name,
+          snapped_core,
+          row_parity,
+          std::set<odb::dbSite*>(flipped_sites.begin(), flipped_sites.end()));
```

Comment:
Why take this as a vector if you'll have to convert it to a set later?

Reply:
Can SWIG bridge std::set? I legitimately don't know. I guess I'll try.

Reply:
I believe so but don't forget `%include <std_set.i>`

Reply:
Turns out it doesn't for Tcl, but for Python it does, but it only links Python lists and tuples to std::set, but crucially not Python set. Super unintuitive gotcha…